src/app/components/UserAssets/Vesting/VestedItem.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Tooltip } from '@blueprintjs/core';
+import { bignumber } from 'mathjs';
+import { Asset } from '../../../../types';
+import { LoadableValue } from '../../LoadableValue';
+import {
+  numberToUSD,
+  weiToNumberFormat,
+} from '../../../../utils/display-text/format';
+import { weiToFixed } from '../../../../utils/blockchain/math-helpers';
+import { translations } from '../../../../locales/i18n';
+import { ActionButton } from '../../Form/ActionButton';
+import { AssetsDictionary } from '../../../../utils/dictionaries/assets-dictionary';
+import { useMaintenance } from '../../../hooks/useMaintenance';
+import { useCachedAssetPrice } from '../../../hooks/trading/useCachedAssetPrice';
+import { FullVesting } from './useListOfUserVestings';
+
+type VestedItemProps = {
+  vesting: FullVesting;
+  onWithdraw: (vesting: FullVesting) => void;
+};
+
+const VestedItem: React.FC<VestedItemProps> = ({ vesting, onWithdraw }) => {
+  const { logoSvg, symbol, decimals } = AssetsDictionary.get(vesting.asset);
+
+  const { t } = useTranslation();
+  const { checkMaintenance, States } = useMaintenance();
+  const withdrawLocked = checkMaintenance(States.WITHDRAW_VESTS);
+  const dollars = useCachedAssetPrice(vesting.asset, Asset.USDT);
+  const dollarValue = useMemo(() => {
+    if ([Asset.USDT, Asset.DOC].includes(vesting.asset)) {
```

Comment:
What about `Asset.XUSD`? I think it should be included in this list.

Reply:
USD rate is actually RUSDT, and doc is aleays == RUSDT, but XUSD is part of amm so it may be slightly different :)

Reply:
ok, I didn't know that :)